osu.Game/Database/OsuDbContext.cs

```diff
@@ -134,7 +134,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<SkinInfo>().HasIndex(b => b.DeletePending);
 
             modelBuilder.Entity<DatabasedKeyBinding>().HasIndex(b => new { b.RulesetID, b.Variant });
-            modelBuilder.Entity<DatabasedKeyBinding>().HasIndex(b => b.IntAction);
```

Comment:
I'd say both indices need to co-exist for now.

Reply:
Will need to add the indexing of StringAction into the migration too